test/pages/product/index.spec.tsx

```diff
@@ -1,18 +1,18 @@
 import { useRouter } from "next/router";
-import SidePanel from "@pages/product/[productId]"; 
+import ProductAppExtension from "@pages/productAppExtension/[productId]"; 
 import { render, screen} from '@test/utils';
 
 jest.mock('@lib/hooks', () => require('@mocks/hooks'));
 jest.mock('next/router', () => ({
     useRouter: jest.fn(),
 }));
 
-describe('SidePanel', () => {
+describe('ProductAppExtension', () => {
     const router = { query: { pid: '1' } };
     useRouter.mockReturnValue(router);
 
     test('renders correctly', async () => {
-        const { container } = render(<SidePanel />);
+        const { container } = render(<ProductAppExtension />);
         // Wait for table to be rendered
         screen.getByText('Basic Information');
```

Comment:
it seems odd that we have this test, but we're not actually awaiting anything

Reply:
updated